portalnet/src/storage.rs

```diff
@@ -546,7 +549,7 @@ impl PortalStorage {
         })?;
         let storage_usage = Self::get_total_size_of_directory_in_bytes(data_dir)?;
         self.metrics
-            .report_total_storage_usage(storage_usage as f64 / 1000.0);
+            .report_total_storage_usage(storage_usage as f64 / BYTES_IN_MB_F64);
```

Comment:
This is my mistake from before: this line duplicates the stat reporting, since it's also reported other times the method is called internally. I remember thinking that I didn't like it being a side effect inside this `get_` method, but must have forgotten to remove it.  Seems like it can be deleted.